webapp/packages/core-utils/src/isSameDay.test.ts

```diff
@@ -0,0 +1,15 @@
+import { isSameDay } from './isSameDay';
+
+describe('isSameDay', () => {
+  it('should be same day', () => {
+    isSameDay(new Date(), new Date());
+    isSameDay(new Date(2020, 1, 1), new Date(2020, 1, 1));
```

Comment:
please add dates with different time (because now we can change isSameDay implementation so it will pass tests)

Reply:
I added some extra checks for time: hour, minutes, seconds and miliseconds